### PR TITLE
update: Add note about SLA and prepare for GA

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
Added note to Dragonfly documentation advising customers to run a PoC with Aiven for Dragonfly before full production deployment. Removed LA tags for GA.
[MA-2419](https://aiven.atlassian.net/browse/MA-2419)
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

[MA-2419]: https://aiven.atlassian.net/browse/MA-2419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by safa-topal — APPROVED** [review]
Lgtm

**Review by hoang-vo — APPROVED** [review]

File: docs/products/dragonfly/howto/migrate-aiven-redis-df-console.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
 The Aiven Console migration tool simplifies the process of migrating
 databases to the Aiven for Dragonfly managed service.
 
+:::note
+Aiven for Dragonfly is fully supported by Aiven's service level agreements (SLAs),
+ensuring its capability to manage production workloads. As the latest addition to
+Aiven services, we recommend initiating a proof of concept (PoC)
+with Aiven for Dragonfly to thoroughly evaluate its capabilities and confirm its
+fit for your production requirements.
+:::
```
Comment (ArthurFlag) [inline docs/products/dragonfly/howto/migrate-aiven-redis-df-console.md:17]: This is repeated 3 times, so let's make it a reusable snippet to keep things clean?

1. Dump the note in `/static/includes/Dragonfly-SLA.md` or similar.
2. Import it where needed:
 
```
import Note from "@site/static/includes/Dragonfly-SLA.md"

<Note/>
```

**Review by ArthurFlag — CHANGES_REQUESTED** [review]
minor suggestion for maintenance

**Review by ArthurFlag — APPROVED** [review]
Thank you!

**Pull request merged**